tests/unit/specs/components/common/__snapshots__/TmSessionExplore.spec.js.snap

```diff
@@ -132,7 +132,7 @@ exports[`TmSessionExplore shows a form to sign in with an address 1`] = `
               class="tm-li-session-title"
             >
               <span>
-                cosmos15ky9d…rm5ctpesxxn9
+                cosmos…gc05sslwesuy
```

Comment:
I think it is better like it was before. Showing four at the beginning, after the `cosmos` prefix, and four at the end.

Like this is much easier to identify imo

Reply:
would be a change though. cosmos…gc05sslwesuy is how we displayed it all the time